ols/app/models/config.py

```diff
@@ -823,8 +819,42 @@ def validate_yaml(self) -> None:
                 "product_docs_index_id is specified but product_docs_index_path is missing"
             )
 
+
+class ReferenceContent(BaseModel):
+    """Reference content configuration."""
+
+    embeddings_model_path: Optional[FilePath] = None
+    indexes: Optional[list[ReferenceContentIndex]] = None
+
+    def __init__(self, data: Optional[dict] = None) -> None:
```

Comment:
Can you try ti just have it as 
```python
class ReferenceContent(BaseModel):
    """Reference content configuration."""

    embeddings_model_path: Optional[FilePath] = None
    indexes: Optional[list[ReferenceContentIndex]] = None
```
without additional methods? Pydantic should handle it on its own.